Cours `BTC-206: Introduction to Bitcoin Inscriptions`

GaloisField2718 opened the pull request:
Remplace la PR #3 sur le cours Ordinals.

GaloisField2718 committed:
Add detailled plan for Bitcoin Inscriptions course

GaloisField2718 committed:
Ajout de l'en-tete dans fr.md

Asi0Flammeus commented:
Je te conseille de mettre ton plan dans un fichier `plan.txt` comme ça tu peux commencer à travailler sur ton fichier `fr.md` ! Sinon ton plan me semble correspondre à ce dont on avait discuté :+1:

GaloisField2718 committed:
Precautions

GaloisField2718 committed:
Merge branch 'DecouvreBitcoin:dev' into dev

GaloisField2718 committed:
Update du cours btc206

GaloisField2718 committed:
Merge branch 'DecouvreBitcoin:dev' into dev

GaloisField2718 committed:
Merge branch 'PlanB-Network:dev' into dev

GaloisField2718 committed:
Test footnotes

GaloisField2718 committed:
btc206: proche de la fin

GaloisField2718 committed:
Merge branch 'PlanB-Network:dev' into dev

GaloisField2718 committed:
btc206: corrections orthographes

GaloisField2718 committed:
Update fr.md

GaloisField2718 committed:
Update fr.md

typo

GaloisField2718 commented:
@Asi0Flammeus 

Le cours me semble bien pour review je pense. Je ne sais pas comment utiliser le traducteur officiel. 
Après traduction je peux le faire vérifier également par des anglophones de l'écosystème. 

Je me tiens à ta disposition et celle des reviewers pour tout retour.

GaloisField2718 committed:
Update fr.md

Mets en note de bas de page un paragraphe de remarque sur la vie de Casey, définit protocole et donne 3 exemples, reformule ce qu'est Ordinals mais reste encore hésitant. A peaufiner

GaloisField2718 committed:
Update fr.md

Premières modifs intro

GaloisField2718 committed:
Update fr.md

Modifs intro

GaloisField2718 committed:
Update fr.md

Typo

GaloisField2718 committed:
réécriture en cours

GaloisField2718 committed:
réécriture en cours

GaloisField2718 committed:
Merge branch 'PlanB-Network:dev' into dev

GaloisField2718 committed:
réécriture en cours

GaloisField2718 committed:
réécriture en cours

GaloisField2718 committed:
Réécriture terminée

GaloisField2718 committed:
Réécriture terminée, typo

GaloisField2718 committed:
Merge branch 'PlanB-Network:dev' into dev

GaloisField2718 committed:
typo

GaloisField2718 committed:
add note fr.md

GaloisField2718 committed:
Merge branch 'PlanB-Network:dev' into dev

Asi0Flammeus commented:
@GaloisField2718 Peux tu me mettre à jour concernant le status de cet PR?
Le contenu est il suffisant pour en faire un cours, ou peut-on l'adapter pour en faire un tutoriel?

GaloisField2718 commented:
Salut @Asi0Flammeus 
Si ca ne convient pas pour être intégré tel quel je n'aurais pas le temps de le modifier encore donc je ferme la PR et si quelqu'un veut le reprendre les documents seront ici. 

Bonne journée.
